internal/datadog/agent.go

```diff
@@ -148,9 +144,6 @@ func (p *TraceAgent) Ingest(ctx context.Context, traces ptrace.Traces) {
 		// ...the call transforms the OTLP Spans into a Datadog payload and sends the result
 		// down the p.pchan channel
 
-		// Stats will be computed for p. Mark the original resource spans to ensure that they don't
-		// get computed twice in case these spans pass through here again.
-		rspans.Resource().Attributes().PutBool(keyStatsComputed, true)
```

Comment:
This will break datadogprocessor if anyone is still using it.

Reply:
We should delete `datadogprocessor` soon. No one should be using that.

Reply:
Let's merge #31037 first then :)